### PR TITLE
sitemap addtion

### PR DESCRIPTION
**Pull request by krishnamangal98**
#854 

## Screenshots
Screenshot (64)

### CODE REVIEW

File: public/sitemap.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+      xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9
+            http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
+
+<url>
+  <loc>https://projectshut.vercel.app/</loc>
+  <lastmod>2023-05-26T14:19:26+00:00</lastmod>
+</url>
+
+
+</urlset>
```
Comment (priyankarpal) [inline public/sitemap.xml:14]: hey @krishnamangal98 we have a few pages such as the `projects` page, and `docs` page so ig we need to add sitemap for those pages as well

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by krishnamangal98** [comment]
I have updated the code.
please check it now

**Review by priyankarpal — CHANGES_REQUESTED** [review]
there are some formatting issues. Please correct with this new one

```<?xml version="1.0" encoding="UTF-8"?>
<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9
                            http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">

  <url>
    <loc>https://projectshut.vercel.app/</loc>
    <lastmod>2023-05-26T14:19:26+00:00</lastmod>
  </url>

  <url>
    <loc>https://projectshut.vercel.app/projects</loc>
    <lastmod>2023-05-26T14:19:26+00:00</lastmod>
  </url>

  <url>
    <loc>https://projectshut.vercel.app/docs</loc>
    <lastmod>2023-05-26T14:19:26+00:00</lastmod>
  </url>

  <url>
    <loc>https://github.com/priyankarpal/ProjectsHut</loc>
    <lastmod>2023-05-26T14:19:26+00:00</lastmod>
  </url>

</urlset>
```

**Comment by krishnamangal98** [comment]
updated.

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**